portal/config/eproms/Organization.json

```diff
@@ -1574,6 +1574,11 @@
                 "code": "en_US",
                 "display": "American English",
                 "system": "urn:ietf:bcp:47"
+              },
+              {
+                "code": "fr_CA",
+                "display": "French Canadian",
```

Comment:
Can you rename that to "Canadian French" to match what's in [Codings.json](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/config/eproms/Coding.json#L6077)?
I think that's the [standardized name](http://www.localeplanet.com/icu/fr-CA/index.html)